src/util/strongPasswordCheck.ts

```diff
@@ -1,22 +1,23 @@
-export default function (password: string) {
-
-
+export default function strongPasswordCheck(password: string) {
   const lowercase = password.match(/[a-z]/g);
   const uppercase = password.match(/[A-Z]/g);
   const number = password.match(/[0-9]/g);
   const specialChar = password.match(
-    /[\!\@\#\$\%\^\&\*\)\(\+\=\.\<\>\{\}\[\]\:\;\'\"\|\~\`\_\-]/g);
+    /[\!\@\#\$\%\^\&\*\)\(\+\=\.\<\>\{\}\[\]\:\;\'\"\|\~\`\_\-]/g
+  );
 
-  //Returnvalue acts as percentage 
+  //Returnvalue acts as percentage
   if (password.length > 4) {
     if (lowercase && uppercase && number && specialChar) {
       return 100;
     }
 
-    if (lowercase && uppercase && number ||
-      lowercase && uppercase && specialChar ||
-      number && specialChar && lowercase ||
-      number && specialChar && uppercase) {
+    if (
+      (lowercase && uppercase && number) ||
+      (lowercase && uppercase && specialChar) ||
+      (number && specialChar && lowercase) ||
+      (number && specialChar && uppercase)
+    ) {
```

Comment:
This change is just related to formatting, as far as I can tell. There is no logic changing here. This is confusing to a reviewer, which is why it's so useful for everyone in a team to configure their editors to use the same automatic formatting. That way, code does not change (reformat) unless it's changing in a meaningful way, which makes it easier to review and look back at later.